presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcConnection.java

```diff
@@ -235,6 +235,17 @@ public void testApplicationName()
         }
     }
 
+    @Test
+    public void testClientTags()
+            throws SQLException
+    {
+        try (Connection connection = createConnection()) {
```

Comment:
This test should be similar to `testExtraCredentials`. We want to test that the connection property is properly propagated to the connection client info. So something like this:
```java
try (Connection connection = createConnection("ClientTags=c2,c3")) {
    assertEquals(clientInfo.getClientInfo("ClientTags"), "c2,c2");
}
```
The `assertConnectionSource` is for validating the `source` property of connections (and we have to test it like that since there's no way to get the final source before it's sent to the server).